scripts/deploy_eks_env.sh

```diff
@@ -110,7 +112,7 @@ export EMAIL_INGRESS
     kustomize edit set image easi-backend=840301899071.dkr.ecr.us-west-2.amazonaws.com/easi-backend:"$GIT_HASH"
     kustomize edit set image easi-frontend=840301899071.dkr.ecr.us-west-2.amazonaws.com/easi-frontend:"$GIT_HASH"
     kustomize edit set image db-migrate=840301899071.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:"$GIT_HASH"
-    kustomize build | envsubst > manifest-easi.yaml
+    kustomize build | envsubst '$EASI_INGRESS,$MINIO_CONSOLE_INGRESS,$EMAIL_INGRESS' > manifest-easi.yaml
```

Comment:
Aren't we relying on more than just these 3 env vars being substituted by `envsubst`? Or is this all that's technically needed to build the manifest?

Reply:
You made me realize here that we actually only need $EASI_INGRESS specifically for the easi-backend_configmap overlay.

Adding a comment here to explicitly note this connection.